packages/utils/src/manifest/types.ts

```diff
@@ -0,0 +1,158 @@
+import { valid as validSemver } from 'semver';
+import {
+  array,
+  enums,
+  Infer,
+  integer,
+  is,
+  literal,
+  object,
+  optional,
+  pattern,
+  refine,
+  size,
+  string,
+  type,
+} from 'superstruct';
+import { assertStruct } from '../assert';
+import { NamespacesStruct } from '../namespace';
+
+/**
+ * A struct for validating a base64 string.
+ *
+ * NOTE: This is basic validation to reduce computation time.
+ */
+export const Base64Struct = pattern(string(), /^[A-Za-z0-9+\\/]*={0,2}$/);
```

Comment:
We could enforce the length here, rather than in the struct(s) below.
```suggestion
export const Base64Struct = size(pattern(string(), /^[A-Za-z0-9+\\/]*={0,2}$/), 44);
```